apps/climatemappedafrica/src/payload/globals/HURUMap/Tutorial.js

```diff
@@ -0,0 +1,122 @@
+import { slateEditor } from "@payloadcms/richtext-slate";
+
+import image from "../../fields/image";
+import richText from "../../fields/richText";
+
+const selectors = [
+  {
+    value: "none",
+    label: "None",
+  },
+  {
+    value: "location-search",
+    label: "Location Search",
+  },
+  {
+    value: "rich-data",
+    label: "Rich Data",
+  },
+  {
+    value: "pin",
+    label: "Pin",
+  },
+];
+
+const Tutorial = {
+  label: "Tutorial",
+  fields: [
+    {
+      name: "enableTutorial",
```

Comment:
Unless Payload rejects them, I think simpler names are better. The global is already called Tutorial
```suggestion
      name: "enabled",
```